### PR TITLE
add InternshipDeleteCommand

### PR DESCRIPTION
**Pull request by HM33-Stu**
Resolves Issue #13 
Resolves Issue #101 

Files Added:
InternshipDeleteCommand
InternshipDeleteCommandParser
InternshipDeleteCommandTest
InternshipDeleteCommandParserTest

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/InternshipDeleteCommand.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.InternshipMessages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.InternshipModel;
+import seedu.address.model.internship.Internship;
+
+/**
+ * Deletes a person identified using it's displayed index from the address book.
```
Comment (CaptainKeqing) [inline src/main/java/seedu/address/logic/commands/InternshipDeleteCommand.java:15]: javadoc comment change to internship

File: src/test/java/seedu/address/logic/commands/InternshipDeleteCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        InternshipDeleteCommand deleteFirstCommandCopy = new InternshipDeleteCommand(INDEX_FIRST_INTERNSHIP);
+        assertTrue(deleteFirstCommand.equals(deleteFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(deleteFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(deleteFirstCommand.equals(null));
+
+        // different person -> returns false
```
Comment (CaptainKeqing) [inline src/test/java/seedu/address/logic/commands/InternshipDeleteCommandTest.java:99]: javadoc person -> internship

**Review by CaptainKeqing — CHANGES_REQUESTED** [review]
lgtm, just 2 small javadoc nits!

File: src/test/java/seedu/address/logic/commands/InternshipDeleteCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+        // different types -> returns false
+        assertFalse(deleteFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(deleteFirstCommand.equals(null));
+
+        // different internship -> returns false
+        assertFalse(deleteFirstCommand.equals(deleteSecondCommand));
+    }
```
Comment (HM33-Stu) [inline src/test/java/seedu/address/logic/commands/InternshipDeleteCommandTest.java:101]: @CaptainKeqing Change made

File: src/main/java/seedu/address/logic/commands/InternshipDeleteCommand.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.InternshipMessages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.InternshipModel;
+import seedu.address.model.internship.Internship;
+
+/**
+ * Deletes the internship identified by the index number used in the displayed internship list.
+ */
```
Comment (HM33-Stu) [inline src/main/java/seedu/address/logic/commands/InternshipDeleteCommand.java:16]: @CaptainKeqing Change made

**Pull request closed without merging**